.vscodeignore

```diff
@@ -17,11 +17,10 @@ src/**
 .gitignore
```

Comment:
I think it is better to do it the other way around - forbidding everything and explicitly including the content we want included in the final vsix. Like we did it in Bridge for Git Explorer.